malariagen_data/anoph/cnv_data.py

```diff
@@ -45,6 +50,15 @@ def __init__(
         self._cache_cnv_coverage_calls: Dict = dict()
         self._cache_cnv_discordant_read_calls: Dict = dict()
 
+    @property
+    def _discordant_read_calls_analysis(self) -> Optional[str]:
+        if self._discordant_read_calls_analysis_override:
```

Comment:
```suggestion
        if isinstance(self._discordant_read_calls_analysis_override, str):
```

Suggest we use `None` to indicate that the analysis version from the config should be used, and the empty string `""` to indicate an override that the original analysis version should be used. 

So here we check to see if the user provided a string, and if so, return it.